### PR TITLE
Work for ts 126

### PR DESCRIPTION
**Pull request by reshmabidikar**
- Updated `snappy-java` to `1.1.10.4`
- Excluded `protobuf` dependency
- Updated logback to `1.3.14` and `slf4j` to `2.0.7`

### CODE REVIEW

**Comment by xsalefter** [comment]
@reshmabidikar Is it possible if we set slf4j to `2.0.11` instead? I see that [dependabot already asking](https://github.com/killbill/killbill-oss-parent/pull/780) for `2.0.9` (meaning if not, soon, we need to update it again), and I just checked that slf4j latest version now is `2.0.11` (I think it is always safe to change patch version).

File: pom.xml [hunk 1]
```diff
@@ -559,6 +559,12 @@
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
                 <version>8.0.30</version>
+                <exclusions>
```
Comment (sbrossie) [inline pom.xml:562]: Re-reading the comment from [TS#126](https://github.com/killbill/technical-support/issues/126): 

```
I believe this is a transitive dependency (included by the [mysql:mysql-connector-java](https://github.com/killbill/killbill-oss-parent/blob/01ec138033c402e0782164170573d977051743f2/pom.xml#L560) dependency). So, if we upgrade mysql:mysql-connector-java to 8.0.33, this should be resolved. However, this needs to be confirmed.
```

Can you remind me why we are suggesting to upgrade to ` 8.0.33` - and if so, why we don't do it?

Comment (reshmabidikar) [reply pom.xml:562]: > Re-reading the comment from [TS#126](https://github.com/killbill/technical-support/issues/126):
> 
> ```
> I believe this is a transitive dependency (included by the [mysql:mysql-connector-java](https://github.com/killbill/killbill-oss-parent/blob/01ec138033c402e0782164170573d977051743f2/pom.xml#L560) dependency). So, if we upgrade mysql:mysql-connector-java to 8.0.33, this should be resolved. However, this needs to be confirmed.
> ```
> 
> Can you remind me why we are suggesting to upgrade to ` 8.0.33` - and if so, why we don't do it?

`protobuf-java` is indeed a transitive dependency and needs the mysql dependency version to be updated to `8.0.33`. This requires changes in the killbill-commons repo too as mentioned [here](https://github.com/killbill/technical-support/issues/126#issuecomment-1899755517). So, I tried out [your suggestion](https://github.com/killbill/technical-support/issues/126#issuecomment-1900858937) of excluding the protobuf-java dependency and that worked too. So,  there is now no need to update mysql to `8.0.33`.

**Comment by reshmabidikar** [comment]
> @reshmabidikar Is it possible if we set slf4j to `2.0.11` instead? I see that [dependabot already asking](https://github.com/killbill/killbill-oss-parent/pull/780) for `2.0.9` (meaning if not, soon, we need to update it again), and I just checked that slf4j latest version now is `2.0.11` (I think it is always safe to change patch version).

Sure, let me give it a try.

**Comment by reshmabidikar** [comment]
@xsalefter, I've updated slf4j to `2.0.9`. 👍

**Pull request merged**